### PR TITLE
Adding another converted pnach.

### PR DESCRIPTION
**Pull request by backgamon**
i hope all are right.
All pnach are checked/tested OK.

### CODE REVIEW

File: patches/SLES-51687_D8A68BBA.pnach [hunk 1]
```diff
@@ -0,0 +1,15 @@
+gametitle=Pitfall - The Lost Expedition (PAL-F) SLES-51687 D8A68BBA
+
+[Widescreen 16:9]
+gsaspectratio=16:9
+author=Arapapa
+comment=Widescreen Hack Converted by Bigdemon
```
Comment (pgert) [inline patches/SLES-51687_D8A68BBA.pnach:6]: I'll take it you mean "ported by" (from another region),
 so then express it so - for clarity.

File: patches/SLES-51989_BF45EFF4.pnach [hunk 2]
```diff
@@ -1,5 +1,14 @@
 gametitle=Wallace & Gromit in Project Zoo (PAL-M) SLES-51989 BF45EFF4
 
+[Widescreen 16:9]
+gsaspectratio=16:9
+author=Bigdemon
+comment=Widescreen hack Conversion
+
+//X-Fov
+//4040013c00088144 01b01846
```
Comment (pgert) [inline patches/SLES-51989_BF45EFF4.pnach:9]: Missing whitespace: "4040013c00088144" -> "4040013c 00088144".

File: patches/SLES-52103_2F1F3758.pnach [hunk 3]
```diff
@@ -0,0 +1,9 @@
+gametitle=Tak and The Power of Juju (PAL-F) SLES-52103 2F1F3758
```
Comment (pgert) [inline patches/SLES-52103_2F1F3758.pnach:1]: "Tak & Le Pouvoir de Juju"
https://wiki.pcsx2.net/Tak_and_the_Power_of_Juju
http://redump.org/disc/26130/
... and according to GameIndex.

File: patches/SLES-52552_BCCAB4C0.pnach [hunk 4]
```diff
@@ -0,0 +1,27 @@
+gametitle=Samurai Warriors (PAL-F) (SLES-52552)
+
+[Widescreen 16:9]
+gsaspectratio=16:9
+author=ElHecht
+comment=Widescreen Hack Converted by Bigdemon
```
Comment (pgert) [inline patches/SLES-52552_BCCAB4C0.pnach:6]: ... ported by

File: patches/SLES-52896_34DA05D2.pnach [hunk 5]
```diff
@@ -0,0 +1,18 @@
+gametitle=Spongebob Squarepants Movie Game (PAL-F) (SLES-52896)
```
Comment (pgert) [inline patches/SLES-52896_34DA05D2.pnach:1]: This disc is the PAL-S version - not the PAL-F:
http://redump.org/disc/26253/
... and according to GameIndex.

File: patches/SLES-52896_34DA05D2.pnach [hunk 6]
```diff
@@ -0,0 +1,18 @@
+gametitle=Spongebob Squarepants Movie Game (PAL-F) (SLES-52896) 
+
+[Widescreen 16:9]
+gsaspectratio=16:9
+author=Bigdemon
```
Comment (pgert) [inline patches/SLES-52896_34DA05D2.pnach:5]: And it was ported by GTAZAZO98:
https://forums.pcsx2.net/Thread-PCSX2-Widescreen-Game-Patches?pid=624538#pid624538

File: patches/SLES-54807_A54496E5.pnach [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+gsaspectratio=16:9
+author=Bigdemon
+comment=Widescreen Hack Conversion
+
+//Zoom
+//003f013c 00608144 000099c4
+patch=1,EE,0040cd60,word,3c013f20 //3c013f00
+
+//Y-Fov
+//4040013c 00008144 803e013c 00088144 02a00046(2nd)
```
Comment (pgert) [inline patches/SLES-54807_A54496E5.pnach:13]: Missing whitespace: "02a00046(2nd)" -> "02a00046 (2nd)".

File: patches/SLES-82003_E7D4DC18.pnach [hunk 8]
```diff
@@ -0,0 +1,12 @@
+gametitle=Summoner (PAL-F) (SLES-82003)
+
+[Widescreen 16:9]
+gsaspectratio=16:9
+author=ElHecht
+comment=Widescreen Hack Converted by Bigdemon
```
Comment (pgert) [inline patches/SLES-82003_E7D4DC18.pnach:6]: ... ported by

File: patches/SLES-52896_34DA05D2.pnach (hunk 5 above)
Comment (backgamon) [reply patches/SLES-52896_34DA05D2.pnach:1]: I made a little mistake but this is not the PAL-S:
http://redump.org/disc/26144/

File: patches/SLES-52896_34DA05D2.pnach (hunk 6 above)
Comment (backgamon) [reply patches/SLES-52896_34DA05D2.pnach:5]: Games share the same CRC.

**Pull request merged**